extensions/wikia/ImageReview/ImageReviewSpecialController.class.php

```diff
@@ -131,50 +76,28 @@ public function index() {
 				'default'					=> ImageReviewStatuses::STATE_UNREVIEWED
 			);
 
-			if ( isset( $do[ $action ] ) ) {
-				$this->imageList = $helper->getImageList( $ts, $do[ $action ], $order );
-				$this->imageCount = $helper->getImageCount();
+			if ( isset( $do[ $this->action ] ) ) {
+				$this->imageList = $this->helper->getImageList( $this->ts, $do[ $this->action ], $this->order );
+				$this->imageCount = $this->helper->getImageCount();
 			} else {
-				$this->imageList = $helper->getImageList( $ts, $do[ 'default' ], $order );
-				$this->imageCount = $helper->getImageCount( 'unreviewed', count( $this->imageList ) );
+				$this->imageList = $this->helper->getImageList( $this->ts, $do[ 'default' ], $this->order );
+				$this->imageCount = $this->helper->getImageCount( 'unreviewed', count( $this->imageList ) );
 			}
 		} else {
-			$this->imageCount = $helper->getImageCount();
+			$this->imageCount = $this->helper->getImageCount();
 		}
 
-		/* SUS-541 / Mix <mix@wikia.com> / scope: the following if block */
-		if ( count( $this->imageList ) < ImageReviewHelper::LIMIT_IMAGES ) {
-			$severity = 'error';
-		} else {
-			$severity = 'success';
-		}
-		WikiaLogger::instance()->debug(
-			'SUS-541',
-			[
-				'severity' => $severity,
-				'imageList' => $this->imageList,
-				'exception' => new Exception
-			]
-		);
-	}
+		$this->setVariables();
 
-	protected function getOrderingMethod() {
-		if ($this->wg->user->isAllowed('imagereviewcontrols')) {
-			$order = (int)$this->getVal('sort', -1);
-			if ($order >= 0) {
-				$this->app->wg->User->setGlobalPreference('imageReviewSort', $order);
-				$this->app->wg->User->saveSettings();
-			}
+		$this->imageCount = array_map(function($number) {
```

Comment:
@sqreek This transformation is needed for frontend to be able to display those values properly. However function operates on integers and stores them in cache as integers to allow offline update of those values easily. I could apply transformation on function output but it is more "backend" function so it should not deal with representation of this data - it is "frontend" task hence this code is in controller.